sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/ResourceTest.java

```diff
@@ -15,7 +16,10 @@
 
   @Test
   void resource() {
-    Attributes attributes = OpenTelemetrySdkAutoConfiguration.getResource().getAttributes();
+    // make sure that the SDK is autoconfigured first regardless of test execution order
+    GlobalOpenTelemetry.get();
```

Comment:
why is this necessary when you're only testing the Resource?

Reply:
No longer applicable, removed.